projects/1.18/assets/construction-wand/constructionwand/lang/zh_cn.json

```
@@ -3,49 +3,68 @@
   "constructionwand.option.lock.horizontal.desc": "在起始方块的前面延伸一行水平方块",
-  "constructionwand.option.lock.vertical": "§a上 / 下",
+  "constructionwand.option.lock.vertical": "§a上/下",
   "constructionwand.option.lock.vertical.desc": "在起始方块的前面延伸一列竖直方块",
-  "constructionwand.option.lock.northsouth": "§6南 / 北",
-  "constructionwand.option.lock.northsouth.desc": "在起始方块的上面，向南 / 北方向延伸一行",
-  "constructionwand.option.lock.eastwest": "§6东 / 西",
-  "constructionwand.option.lock.eastwest.desc": "在起始方块的上面，向东 / 西方向延伸一行",
+  "constructionwand.option.lock.northsouth": "§6南/北",
+  "constructionwand.option.lock.northsouth.desc": "在起始方块的上面，向南/北方向延伸一行",
+  "constructionwand.option.lock.eastwest": "§6东/西",
+  "constructionwand.option.lock.eastwest.desc": "在起始方块的上面，向东/西方向延伸一行",
   "constructionwand.option.lock.nolock": "§c无",
   "constructionwand.option.lock.nolock.desc": "从起始方块向任意一面延伸",
+
   "constructionwand.option.direction": "方向：",
   "constructionwand.option.direction.target": "§6目标",
   "constructionwand.option.direction.target.desc": "放置的方块方向与目标方块的方向相同",
   "constructionwand.option.direction.player": "§a玩家",
   "constructionwand.option.direction.player.desc": "放置的方块面向玩家",
+
   "constructionwand.option.replace": "替换：",
   "constructionwand.option.replace.yes": "§a是",
   "constructionwand.option.replace.yes.desc": "替换某些方块，例如流体、雪、高草丛",
   "constructionwand.option.replace.no": "§c否",
   "constructionwand.option.replace.no.desc": "不替换方块",
+
   "constructionwand.option.match": "匹配：",
   "constructionwand.option.match.exact": "§a精确",
   "constructionwand.option.match.exact.desc": "只放置完全相同的方块",
   "constructionwand.option.match.similar": "§6模糊",
   "constructionwand.option.match.similar.desc": "相似的方块被认为是相同的（草方块 / 泥土类型）",
   "constructionwand.option.match.any": "§c任意",
   "constructionwand.option.match.any.desc": "放置任意方块",
+
   "constructionwand.option.random": "随机：",
   "constructionwand.option.random.yes": "§a是",
   "constructionwand.option.random.yes.desc": "随机放置快捷栏中的方块",
   "constructionwand.option.random.no": "§c否",
   "constructionwand.option.random.no.desc": "不会随机放置方块",
-  "stat.constructionwand.use_wand": "放置方块需要使用手杖"
+
+  "constructionwand.description.wand": "%1$s可以在你面前的建筑物一侧放置多达%2$d方块，并记录最近的%3$s。\n\n按住%5$s并滚动以改变放置限制（水平、垂直、南/北、东/西、无锁定）。\n\n用%6$s§9+右键§0打开选项屏幕。\n\n§5§n撤销§0§r\n按住§9潜行+§0%4$s并看向你最后放置的方块，周围会显示绿色边框。§9潜行+§0%4$s§9+右键点击§0其中任何一个就会撤销操作，并向你返还所有物品。如果你使用了破坏核心，它将恢复这些方块。\n\n§5§n容器§0§r\n潜影盒、捆绑物和其他模组的许多容器都可以为手杖提供方块。\n\n§5§n副手优先§0§r\n如果你的副手有方块，则会优先使用副手的方块，而不是背包内你正在看的方块。",
+  "constructionwand.description.durability.limited": "%d方块",
+  "constructionwand.description.durability.unlimited": "无限方块",
+  "constructionwand.description.key.sneak": "潜行",
+  "constructionwand.description.key.sneak_opt": "潜行+%s",
+  "constructionwand.description.core": "§5§n安装§0§r\n把你的新核心和你的手杖一起放在一个合成格里来安装它。要在不同的核心之间进行切换，请按住%s并用你的手杖左键点击空气处，或使用选项屏幕。",
+  "constructionwand.description.core_angel": "天使核心在你面前的方块（或整行方块）的对面放置一个方块。最大距离取决于手杖等级。右击空气可以在半空中放置一个方块。要做到这一点，你需要把你想放置的方块放在你的副手。",

```

```suggestion
  "constructionwand.description.core_angel": "天使核心在你面前的方块（或整行方块）的对面放置方块。最大距离取决于手杖等级。右击空气可以在半空中放置一个方块。要做到这一点，你需要把你想放置的方块放在你的副手。",
```
原文：**The angel core places a block on the opposite side of the block (or row of blocks) you are facing. Maximum distance depends on wand tier. Right click empty space to place a block in midair. To do that, you'll need to have the block you want to place in your offhand.**

---

重点：`a block`，而且确实是放一个